java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -104,4 +104,6 @@ private static void loadNativeLibrary() {
 
     // Closing needs to be done here when initialization fails and by VWBase
     static native void closeInstance(long nativePointer);
+
+    static native void saveModel(long nativePointer, String filename);
```

Comment:
I think the native method should be private the one exposed should be `java.io.File.`